src/client/dfs/duns.c

```diff
@@ -441,17 +444,32 @@ duns_resolve_path(const char *path, struct duns_attr_t *attr)
 
 	/** no match for direct format, do the UNS fs check */
 
+#ifdef LUSTRE_INCLUDE
```

Comment:
I have the same question as above like Johann; why not make this change non conditional to lustre, so also for dfuse and other FS?